content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -12,27 +12,24 @@ for storage.
 Here is a summary of the process:
 
 1. You, as cluster administrator, create a PersistentVolume backed by physical
-storage. You do not associate the volume with any Pod.
+   storage. You do not associate the volume with any Pod.
```

Comment:
I don't understand why there are so many reformattings in this page...

Reply:
I think the reason for all the formatting is my code editor (using VS code).
I tried to change it as I was before but not able to.

Reply:
maybe you should use notepad for markdown, or tweak your settings :)
(it makes reviewing more difficult)

Reply:
I have made the change can you review it.

Reply:
This will be much easier to review if you either:
- omit the reformatting (start again, and configure your editor not to change the parts of the page you're not updating, or use `git add -p` to select which changes you want to include)
- use 2 commits: the first commit to reformat the page with no other changes, and the second commit to make the changes.

We might be able to help you learn what's needed here. I'm afraid SIG Docs is a bit light on people's time right now, so there's a good chance we can't help you one on one. SIG Contributor Experience might have some links to guides on how to use Git and GitHub; I'm not sure.

As this stands, it's not at all easy to make sense of the proposed changes. We couldn't accept the fixes as they are.